### DISCUSSION TITLE
UTM M1 Mac Raspberry Pi issues tried Arm64 & x86_64

### DISCUSSION THREAD

**Discussion by AJTumbleweed** [utmapp/UTM#4970] [UTM Help (macOS)]
I am trying to run Raspberry Pi on my M1 mac. 

When I use Arm64 I am stuck on startup.nsh. Like in the link below

https://github.com/utmapp/UTM/discussions/3470

<img width="796" alt="Screen Shot 2023-01-26 at 6 35 18 PM" src="https://user-images.githubusercontent.com/56708386/214974721-54de9c99-7132-437f-b130-821e01c0e191.png">

<img width="895" alt="Screen Shot 2023-01-26 at 6 36 51 PM" src="https://user-images.githubusercontent.com/56708386/214974784-b6709bfd-7f4c-4ea5-8599-45b6052bedf7.png">

Their solution was to change architecture from Arm64 to x86_64

So I try that and get 

<img width="795" alt="Screen Shot 2023-01-26 at 6 37 20 PM" src="https://user-images.githubusercontent.com/56708386/214974823-4dba0aee-7145-417c-b6a9-ab8ada984293.png">

<img width="891" alt="Screen Shot 2023-01-26 at 6 36 15 PM" src="https://user-images.githubusercontent.com/56708386/214974817-f70fb6cf-3f51-4ed2-bb97-6d582b53d88f.png">

Their solution was to change architecture from x86_64 to Arm64

https://github.com/utmapp/UTM/discussions/3470

I would greatly appreciate any help!

**Comment by cukabeka** [comment]
same here. I can't find any config yet to start into the raspiOS image within UTM.

**Comment by qoopen0815** [comment]
anyone solved ?
i have same issue too.

**Comment by gxc1998github** [comment]
same here too

**Comment by daza85** [comment]
Anyone got this working?

**Comment by sly74fr** [comment]
Same here also...

**Comment by Pikachu92882828** [comment]
you need to emulate Raspberry Pi architecture
